source/_integrations/honeywell.markdown

```diff
@@ -3,30 +3,27 @@ title: Honeywell Total Connect Comfort (US)
 description: Instructions on how to integrate Honeywell thermostats within Home Assistant.
 ha_category:
   - Climate
+  - Sensor
 ha_release: pre 0.7
 ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@rdfurman'
 ha_domain: honeywell
 ha_platforms:
   - climate
+  - sensor
 ---
 
-The `honeywell` climate platform integrates Home Assistant with _US-based_ [Honeywell Total Connect Comfort (TCC)](https://mytotalconnectcomfort.com/portal/) climate systems.
+The `honeywell` integration integrates Home Assistant with _US-based_ [Honeywell Total Connect Comfort (TCC)](https://mytotalconnectcomfort.com/portal/) climate systems.
 
 It uses the [somecomfort](https://github.com/kk7ds/somecomfort) client library. It does not support the home security functionality of TCC.
 
 If your system is compatible with this integration, then you will be able access it via [https://mytotalconnectcomfort.com/portal/](https://mytotalconnectcomfort.com/portal/) (note the `/portal/`).
 
-## Configuration
-
-To set up this integration, add the following to the `climate:` section of your `configuration.yaml` file:
+{% include integrations/config_flow.md %}
 
-```yaml
-climate:
-# Example configuration.yaml entry
-  - platform: honeywell
-    username: YOUR_USERNAME
-    password: YOUR_PASSWORD
-```
+## Configuration
```

Comment:
This whole section, including all configuration variables can be removed as integration using the configuration flow, should deprecated YAML support.